source/_integrations/modbus.markdown

```diff
@@ -339,10 +339,16 @@ modbus:
 
 modbus entities are grouped below each modbus communication entry.
 
-All modbus entities have the following parameters:
+**REMARK** Each modbus device must have at least 1 entity defined.
+
+Experience have shown that a modbus configuration without entities leads to instability issues, which in worst case impact HA negatively.
```

Comment:
```suggestion
Experience has shown that a modbus configuration without entities leads to instability issues.
```
Grammar correction and suggest removing last part of a sentence which seems redundant.